cockpit/src/subscriptions-client.js

```diff
@@ -60,6 +60,11 @@ function needRender() {
     client.dispatchEvent(ev);
 }
 
+/* Utility function to check if an object has a specific property */
+function hasProperty(object, property) {
+  return Object.prototype.hasOwnProperty.call(object, property);
+}
```

Comment:
Is it really necessary to have special function for this testing? This function is used only on one place in the code.